faster_sam/cloudformation.py

```diff
@@ -205,7 +187,7 @@ def include_files(self):
 
     def load(self, template: Optional[str] = None) -> Dict[str, Any]:
         """
-        Loads the CloudFormation template from the specified path.
+        Loads the CloudFormation template file from the specified path or the find one.
```

Comment:
Sugestão para a descrição deste método:

_Reads CloudFormation template file from the disk and convert it to a dictionary._

_If the template argument is not set it is assumed an YAML file named `template` exists in the current directory._